include/unifex/let_error.hpp

```diff
@@ -256,15 +256,15 @@ class _op<Source, Func, Receiver>::type final {
   using final_receiver = final_receiver_type<Source, Func, Receiver, Error>;
 
 public:
-  template <typename Func2, typename Receiver2>
-  explicit type(Source&& source, Func2&& func, Receiver2&& dest) noexcept(
+  template <typename Source2, typename Func2, typename Receiver2>
+  explicit type(Source2&& source, Func2&& func, Receiver2&& dest) noexcept(
```

Comment:
By a similar argument to the above, in `let_done.hpp`, I think that this expresses more generality than is possible given the change below, with the same curiosity about `clang-tidy`'s attitude towards using `forward` with a class template parameter.